B2C-Tfp-Issuer.md

```diff
@@ -0,0 +1,14 @@
+# Azure AD B2C issuer claim
+
+Azure AD B2C user flows can be customized to emit issuer claim in two formats: `https://<domain>/<tenant>/v2.0/` and `https://<domain>/tfp/<tenant>/<userFlow>/v2.0/`. Originally `tfp` (which stands for Trust Framework Policy) in the URI was used to differentiate between Azure AD and Azure AD B2C . Currently the B2C team is working to streamline the experience and remove `tfp`. Therefore, the current guidance from Azure B2C team is to use `https://<domain>/<tenant>/v2.0/` format. 
+
+### Creating Azure B2C user flow
```

Comment:
i'm not sure why we need this in our wiki, as it's in the b2c docs. plus user flow and policy are actually different, but we are using the term user flow to cover both, so i think this could lead to more confusion.

Reply:
I'll remove this section.

I thought user flow is just a predefined policy, right?

Reply:
you can have [custom policies](https://docs.microsoft.com/en-us/azure/active-directory-b2c/custom-policy-overview#comparing-user-flows-and-custom-policies), so we don't distinguish between the two, but b2c does.